exonum-java-binding-core/src/main/java/com/exonum/binding/service/Service.java

```diff
@@ -118,4 +118,12 @@
    * @param router a router responsible for handling requests to this service
    */
   void createPublicApiHandlers(Node node, Router router);
+
+  /**
+   * Handles block commit event. This handler is invoked for each service after commit of the block.
```

Comment:
...This handler is a callback method which is invoked by the blockchain? after each block commit.